taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)
+
+
+class DriverUpdateForm(forms.ModelForm):
+    class Meta:
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=Driver.objects.all(),
+        widget=forms.CheckboxSelectMultiple()
+    )
+    fields = ("model", "manufacturer", "drivers")
+
+    class Meta:
+        model = Car
+        fields = "__all__"
+
+
+class AddCarForm(forms.ModelForm):
+    assign_me_to_this_car = forms.BooleanField(
+        required=False,
+        widget=forms.CheckboxInput(attrs={"class": "form-check-input", })
+    )
+    remove_me_from_this_car = forms.BooleanField(
+        required=False,
+        widget=forms.CheckboxInput(attrs={"class": "form-check-input", })
+    )
+
+    class Meta:
+        model = Car
+        fields = ("assign_me_to_this_car", "remove_me_from_this_car")

```

Comment:
The `fields` attribute in the `AddCarForm` class only includes the boolean fields `assign_me_to_this_car` and `remove_me_from_this_car`. If you intend to include other fields from the `Car` model, they should be added here.